modules/ROOT/pages/exchange1-faq.adoc

```diff
@@ -68,7 +68,7 @@ Package Explorer and click Anypoint Platform > Configure > Anypoint Studio > Aut
 
 == How can users view the content of a business group?
 
-Users must be assigned to the Exchange Viewer role. Users can view the master organization without being assigned to the Exchange Viewer role, but not business groups. If users are invited to a business group, they won't see other business groups until assigned Roles access.
+Users must be assigned to the Exchange Viewer role. Users can view the root organization without being assigned to the Exchange Viewer role, but not business groups. If users are invited to a business group, they won't see other business groups until assigned Roles access.
```

Comment:
The question is about the business group, so I'd omit the distraction of root access:
```suggestion
Users must be assigned to the Exchange Viewer role to view business groups. If users are invited to a business group, they won't see other business groups until assigned Roles access.
```
Is this question and answer specific to users viewing their own group or other groups? What is "Roles" access? Kinda' confusing in the context of the Exchange Viewer role....

Reply:
Yes, this page is all legacy content about Exchange 1, which is in the end-of-life process, and uses different systems for access. Fixed.